src/Observable.ts

```diff
@@ -141,7 +141,7 @@ export class Observable<T> implements Subscribable<T> {
         } else {
           // if there is NO subscription, then we're getting a nexted
           // value synchronously during subscription. We can just call it.
-          // If it errors, Observable's `subscribe` imple will ensure the
+          // If it errors, Observable's `subscribe` imply will ensure the
```

Comment:
oh, sorry, thought you wrote those @staltz . @JayKan - would you mind to update PR to remove those words?